src/script/cpu-map.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+# !
+# ! Usage: ./cpu-map.sh -p <process_PIDs comma separated> -n <process_Name> -g (optional) <by-group>
+# !
+# ! Set the CPU affinity for a list of running processes
+# ! Ex.: ./cpu-map.sh -n scylla
+# !      ./cpu-map.sh -n scylla -g
+# !      ./cpu-map.sh -p "1362, 1049"
+# !      ./cpu-map.sh -p 32 -g
+
+
+########################################################
+# The following must be valid cpu ranges for taskset
+
+# Run with 4 cores on SV1, no hyperthreading
+declare -A thr_grp_map=([alien-store-tp]="4-7" [rocksdb]="4-7" [bstore]="4-7" [cfin]="4-7" )
+free_cpu_avail="4-7"
+# Regex to define the group of threads we want to set their affinity:
+proc_group_re="alien-store-tp|rocksdb|bstore|cfin"
+
+# Regex to ignore:
+proc_ignore_re="crimson-osd|reactor|log|syscall"
```

Comment:
Is the intention for the user to uncomment one of these sections depending on what they are doing?